base-theme/layouts/partials/footer.html

```diff
@@ -1,2 +1,53 @@
-<footer>
+<footer id="footer_container" class="fluid_container column-below-medium">
+    <section class="columns start">
+      <div class>
+        <a id="mit_logo" href="https://www.mit.edu/">
+          <img src="/images/mit-logo-red.svg">
+        </a>
+        <a id="open_learning_logo"  href="https://openlearning.mit.edu/" class="pl-5">
+          <img src= "/images/mit-ol.png">
+        </a>
+      </div>
+      <div class="about-courseware">
+        <h5>
+          ABOUT MIT OPENCOURSEWARE
+        </h5>
+        <p>
+          MIT OpenCourseWare is an online publication of materials from over 2,500
+          MIT courses, freely sharing knowledge with learners and educators
+          around the world. <a href="/about">Learn more</a> »
+        </p>
+      </div>
+      <div class="large-and-above-only">
+        <h5>PROUD MEMBER OF : <a href="https://www.oeglobal.org/"><img class="oeg_logo" src="/images/oeg_logo.gif"></a></h5>
+      </div>
+    </section>
+    <section class="columns end">
+      <div class="links_section">
+        <ul>
+          <li><a href="https://accessibility.mit.edu">ACCESSIBILITY</a></li>
+          <li><a href="https://creativecommons.org/licenses/by-nc-sa/4.0/">Creative Commons License</a></li>
+          <li><a href="https://ocw.mit.edu/terms/">Terms and Conditions</a></li>
+        </ul>
+      </div>
+      <div class="horizontal_list">
+        <ul>
+          <li><a href="https://www.facebook.com/MITOCW/"><i class="material-icons blue600">facebook</i></a></li>
```

Comment:
just wondering if "blue600" is a correct convention for a CSS class. 
ps: Is it a customized class? As I couldn't find it